src/components/header/modals/ModalNetwork.vue

```diff
@@ -238,6 +238,14 @@ export default defineComponent({
       { immediate: true }
     );
 
+    const windowHeight = ref<number>(window.innerHeight);
```

Comment:
![bandicam 2022-05-01 13-06-03-269](https://user-images.githubusercontent.com/40356749/166143319-e21ec890-913d-432c-843a-5a8d620fefb2.jpg)
not sure if it's related, but I'm getting some style errors

Reply:
Fixed :)